### ISSUE TITLE
items not visible if renaming field of collection that has entries

### ISSUE THREAD

**Issue by raffaelj** [agentejo/cockpit#775]
If have a collection and change a field name (for example category instead of cagory). If the collection has entries, a new field "category" without items is visible. The old field "cagory" is still present if I export the data and if I change the field name back or add anew field "cagory", the items of "cagory" are visible again.

I know, I could use the label, but if my field name has a typo, it feels wrong to keep it.

Expected behavior:

I change the field name and the change is populated to all entries. --> maybe with an info field "Maybe you have to change your API requests or application logic"

or:

The field name is readonly if the collection has entries. --> maybe with an info field "Contains entries, changing would break some internal logic"

**Comment by gryphonmyers** [comment]
I agree with the sentiment that right now it feels a bit too loose and messy. You can very easily get cockpit to lose and leave stray data all over the place. It of course opens up a whole can of worms if you try to keep the database in sync when changes like this are made to the schema though. Interesting idea re: making it readonly, though that seems pretty limiting. 

@aheinze do you have any plans or thoughts about this problem?
